### ISSUE TITLE
GUI not working with R 3.5.x

### ISSUE THREAD

**Issue by aalfons** [VLDrenth/robmedShiny#7]
I tried running the GUI with R version 3.5.3. So far, I have not been able to install package `flextable`. This might be an issue with my system, or the package may not work with this older version of `R`. I have not yet sorted this out. It would be good if someone else will try this.

Regardless, if I uncomment the line loading package `flextable` (which should only mean that the button for exporting the table should not work), the GUI does not start and I get this warning message:

```
Warning: Error in strsplit: non-character argument
  3: shiny::runApp
  2: run_shiny_app [/Users/andreas/Documents/Research/robmedShiny/R/run_shiny_app.R#97]
  1: start_GUI [/Users/andreas/Documents/Research/robmedShiny/R/run_shiny_app.R#104]
```

**Comment by AuroreAA** [comment]
I also tried running the GUI with R version 3.5.3. and it is not working for me either.

**Comment by VLDrenth** [comment]
I am currently running with R 3.5.0 with flextable installed so it should probabaly work for 3.5.3 as well, what issue do you come across in installing flextable? In any case, I have now resolved the bug related to strsplit.

**Comment by VLDrenth** [comment]
The original issue had been solved, though the solution in #40 apparently does not work in version 3.5.0. As this is more related to that issue, I am closing this and reference the problem in #40.
